### PR TITLE
fix(playground): reflect spec changes

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

Add `kotlin` playground to the CLI with `yarn docker playground kotlin search`.

Also fix a bunch of issues in the playground, migration guide and example because the specs changed a while back.

### CODE REVIEW

File: playground/kotlin/src/main/kotlin/com/algolia/playground/Insights.kt [hunk 1]
```diff
@@ -19,10 +19,10 @@ suspend fun main() {
     )
     val indexName = dotenv["SEARCH_INDEX"]
 
-    val events = InsightEvents(
+    val events = InsightsEvents(
         events = listOf(
-            InsightEvent(
-                eventType = EventType.Click,
+            EventsItems.ClickedObjectIDs(
+                eventType = ClickEvent.Click,
```
Comment (shortcuts) [inline playground/kotlin/src/main/kotlin/com/algolia/playground/Insights.kt:25]: EventType felt better 🤔

File: playground/kotlin/bin/main/com/algolia/playground/AbTesting.kt
Comment (shortcuts) [inline playground/kotlin/bin/main/com/algolia/playground/AbTesting.kt:1]: is the gitignore up to date?

**Review by shortcuts — COMMENTED** [review]
lovely!

File: playground/kotlin/src/main/kotlin/com/algolia/playground/Insights.kt (hunk 1 above)
Comment (millotp) [reply playground/kotlin/src/main/kotlin/com/algolia/playground/Insights.kt:25]: here each enum has 1 option, so you can't make a mistake. In typescript it's just a string so it's nice

Comment (shortcuts) [reply playground/kotlin/src/main/kotlin/com/algolia/playground/Insights.kt:25]: yup I was talking in term of naming, ClickEvent.Click is redundant IMO

File: playground/kotlin/bin/main/com/algolia/playground/AbTesting.kt
Comment (millotp) [reply playground/kotlin/bin/main/com/algolia/playground/AbTesting.kt:1]: I don't know who generates those files (might be IntelliJ), but here running with the cli it's fine, I can add those files to gitignore yes

**Review by shortcuts — APPROVED** [review]

**Pull request merged**